services/frontend/pkg/revaconfig/config.go

```diff
@@ -301,3 +317,34 @@ func FrontendConfigFromStruct(cfg *config.Config) (map[string]interface{}, error
 		},
 	}, nil
 }
+
+func readMultilineFile(path string) (interface{}, error) {
+	if !fileExists(path) {
+		path = filepath.Join(defaults.BaseConfigPath(), path)
+	}
+	file, err := os.Open(path)
+	if err != nil {
+		return nil, err
+	}
+	defer file.Close()
+	scanner := bufio.NewScanner(file)
+	data := make(map[string]struct{})
+	for scanner.Scan() {
+		line := scanner.Text()
+		if line != "" {
+			data[line] = struct{}{}
+		}
+	}
+	return data, err
+}
+
+func fileExists(path string) bool {
+	info, err := os.Stat(path)
+	if os.IsNotExist(err) {
+		return false
+	}
```

Comment:
Please also return false `if err != nil` just to be on the safe side

Reply:
If we return false when error it doesn't make sense to check  `os.IsNotExist(err)` before.